src/hooks/useEncounterRole.tsx

```diff
@@ -0,0 +1,15 @@
+import { OpenmrsResource, openmrsFetch } from '@openmrs/esm-framework';
+import useSWRImmutable from 'swr/immutable';
+
+export function useEncounterRole() {
+  const { data, error, isLoading } = useSWRImmutable<{ data: { results: Array<OpenmrsResource> } }, Error>(
+    '/ws/rest/v1/encounterrole?v=full',
+    openmrsFetch,
+  );
+  const clinicalEncounterRole = data?.data.results.find(encounterRole => encounterRole.display === 'Clinician');
```

Comment:
I could be wrong, but hypothetically the `display` property is a translatable string by the backend; In such scenarios, the above find operation will be broken. I would use the name property.